ntbs-service/Pages/Shared/_NotificationLayout.cshtml

```diff
@@ -18,123 +15,16 @@
-                                Social Risk Factors
-                            </a>
-                        </li>
-                        <li class="@("contents-list-item " + ((bool)ViewData["IsTravelPage"] ? "contents-list-item--current" : ""))">
-                            <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="@RouteHelper.GetNotificationPath(NotificationSubPaths.EditTravel, Model.NotificationId)">
-                                Travel/visitor history
-                            </a>
-                        </li>
-                        <li class="@("contents-list-item " + ((bool)ViewData["IsComorbiditiesPage"] ? "contents-list-item--current" : ""))">
-                            <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="@RouteHelper.GetNotificationPath(NotificationSubPaths.EditComorbidities, Model.NotificationId)">
-                                Co-morbidities
-                            </a>
-                        </li>
-                        <li class="@("contents-list-item " + ((bool)ViewData["IsImmunosuppressionPage"] ? "contents-list-item--current" : ""))">
-                            <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="@RouteHelper.GetNotificationPath(NotificationSubPaths.EditImmunosuppression, Model.NotificationId)">
-                                Immunosuppression
-                            </a>
-                        </li>
-                        <li class="@("contents-list-item " + ((bool)ViewData["IsPreviousHistoryPage"] ? "contents-list-item--current" : ""))">
-                            <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="@RouteHelper.GetNotificationPath(NotificationSubPaths.EditPreviousHistory, Model.NotificationId)">
-                                Previous History
-                            </a>
-                        </li>
-                    </ul>
-                </nhs-grid-column>
-            }
-            <nhs-grid-column grid-column-width=@mainGridWidth>
-                <input type="hidden" asp-for="NotificationId" />
-                @RenderBody()
-            </nhs-grid-column>
-            <nhs-grid-column grid-column-width="Full">
-                <div class="notification-page-end @endClass"> </div>
-            </nhs-grid-column>
-        </nhs-grid-row>
-    </form>
+    <nhs-grid-row>
+        <nhs-grid-column grid-column-width="Full">
+            <div class="notification-banner-container">
+                <partial name="../Shared/_NotificationBanner.cshtml" model="@Model.NotificationBannerModel" />
+                <div class="notification-banner-triangle @notificationStatusTriangleCssClass"> </div>
+            </div>
+        </nhs-grid-column>
+        @RenderBody()

```

Comment:
I think it's a bit dangerous to delegate to body between `row` and `column`, as they are very tightly coupled tags and now one has to look across multiple files to see if they are nested correctly.
I don't think we're gaining much from this line being inside the row? If so, I would suggest we move it down a line, to be out of it's the row's scope and let each page define its row as if needed

Reply:
Was trying on the whole to limit changes to individual pages - will get this fix propagated through